plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -146,17 +146,47 @@ const generateNextReviewAssignments = async ({
   console.log('Next level reviewers', nextLevelReviewers)
   const reviewAssignments: ReviewAssignmentObject = {}
 
+  // Check if other reviewAssignmet is already assigned to create new ones LOCKED
+  const existingReviewAssignments: ExistingReviewAssignment[] =
+    await db.getExistingReviewAssignments(applicationId, stageNumber, nextReviewLevel)
+
+  const getNewOrExistingAssignmentStatus = (
+    userId: number,
+    canMakeFinalDecision: boolean,
+    canSelfAssign: boolean
+  ) => {
+    // Temporarly final decision shouldn't be locked if there are other reviewAssignemt assigned
```

Comment:
Is it "temporarily"? I thought it would be permanent (at least as far as this action is concerned)

Reply:
hahahha yes, it would for now - I was thinking I would be working with issue #476 and check if this would be changed or just rephrase this comment in case not. Updating to "temporarily" anyways.

Reply:
What I mean was why are you saying "temporarily" at all? Shouldn't it just say "Final decision shouldn't be locked if there are other reviewAssignemt assigned" -- what's temporary about it?